### PR TITLE
Cpu features

### PR DESCRIPTION
**Pull request by anton-malakhov**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by anton-malakhov** [comment]
@conda-forge/help-c-cpp please help with osx build. It looks like it cannot extract the archive. It worked in fef4fa0 but stopped working in later commits when I changed the version to new one.

File: recipes/cpu_features/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "cpu_features" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "2feca41fe51d32f53d22d7fb75e4b6ef5cab70db0ff2620259bbfc8f1a9cd887" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.src.tar.xz
```
Comment (chrisburr) [inline recipes/cpu_features/meta.yaml]: I’m not convinced it’s causing the macOS failure but this line isn’t needed
```suggestion
```

File: recipes/cpu_features/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'A cross platform C99 library to get cpu features at runtime'
+  dev_url: https://github.com/google/cpu_features
+
+extra:
+  recipe-maintainers:
+    - anton-malakhov
+    - aregm
```
Comment (chrisburr) [inline recipes/cpu_features/meta.yaml]: @aregm are you happy being listed as a maintainer of this recipe?

Comment (aregm) [reply recipes/cpu_features/meta.yaml]: > @aregm are you happy being listed as a maintainer of this recipe?

Yes, confirmed! Thanks.

**Comment by anton-malakhov** [comment]
Thanks Chris!! Apparently with regard to `fn:`, there is a bug in the infrastructure and/or with conda-build itself.. Appveyor's behavior is strange as well - it's either intermittent problem or `%SRC_DIR%`  did not work as expected.. does it execute builds for different vc versions in not fully isolated environments?

**Pull request merged**